examples/cmd/vppagent-firewall-nse/interface.go

```diff
@@ -138,18 +138,21 @@ func newVppAgentXConnComposite(configuration *common.NSConfiguration) *vppAgentX
 		crossConnects:    make(map[string]crossConnectStruct),
 		workspace:        configuration.Workspace,
 	}
-	newVppAgentXConnComposite.SetSelf(newVppAgentXConnComposite)
 	newVppAgentXConnComposite.reset()
 
 	return newVppAgentXConnComposite
 }
 
 type vppAgentAclComposite struct {
-	endpoint.BaseCompositeEndpoint
+	endpoint.ChainedImpl
 	vppAgentEndpoint string
 	aclRules         map[string]string
 }
 
+func (vac *vppAgentAclComposite) GetOpaque(interface{}) interface{} {
```

Comment:
Why not define these in ChainedImpl? DO we need to repeate this empty implemntation everywhere?

Reply:
Good idea, done